pkg/chaosctl/common/common.go

```diff
@@ -34,9 +34,10 @@ import (
 
 	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
 	ctrlconfig "github.com/chaos-mesh/chaos-mesh/controllers/config"
+	daemonClient "github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/client"
 	"github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb"
 	"github.com/chaos-mesh/chaos-mesh/pkg/portforward"
-	"github.com/chaos-mesh/chaos-mesh/pkg/utils"
+	sele "github.com/chaos-mesh/chaos-mesh/pkg/selector"
```

Comment:
Same request for using `selector` instead of `sele`

Reply:
Fixed